package.json

```diff
@@ -4,6 +4,8 @@
   "appBundleId": "org.owasp.threatdragon",
   "private": true,
   "scripts": {
+    "predev:vue": "node td.vue/src/plugins/generate-cornucopia.js",
```

Comment:
How are scripts `predev:vue` and `prestart:desktop` being run?
These are duplicates of the `td.vue` scripts and which is where they should be located
If these are not necessary they could be deleted and there would be no change to this file and also no change to `package-lock.json`, reducing the size of the PR